.github/workflows/ci.yml

```diff
@@ -6,18 +6,27 @@ on:
 
 env:
   COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   COVERAGE: '0'
   SYMFONY_DEPRECATIONS_HELPER: disabled=1
 
 jobs:
   commitlint:
     runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          configFile: .commitlintrc
-      - uses: wagoid/commitlint-github-action@v5
+      - name: Run commitlint
+        run: | 
+            commit=$(gh api \
```

Comment:
Did you check in `${{ github.event_path }}` var? It's a path of the pull_request JSON data. You may parse it using jq, instead of requesting the API